### PR TITLE
Add Track2 tests

### PR DESCRIPTION
**Pull request by HarveyLink**
### List
 - Compute
    - ManageVirtualMachine
    - CreateVirtualMachinesInParallel
    - ManageVirtualMachineExtension
 - EventHub
    - ManageEventHub
    - ManageEventHubEvents
    - ManageEventHubGeoDisasterRecovery
 - KeyVault
    - ManageKeyVault
 - Network
    - ManageVirtualNetworkAsync
    - ManageIPAddress
 - Resources
    - DeployUsingARMTemplate
    - ManageResource
    - ManageResourceGroup

### CODE REVIEW

File: samples/Helpers/Utilities.cs [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+
+        public static void Log()
+        {
+            Utilities.Log("");
+        }
+
+        public static string GetArmTemplate(string templateFileName)
+        {
+            //var adminUsername = "tirekicker";
+            //var adminPassword = "12NewPA$$w0rd!";
```
Comment (erich-wang) [inline samples/Helpers/Utilities.cs]: Please remove these two lines

File: samples/compute/create-virtual-machines-in-parallel/Program.cs [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    {
+        /**
+         * Azure compute sample for creating multiple virtual machines in parallel.
+         *  - Define 1 virtual network per region
+         *  - Define 1 storage account per region
+         *  - Create 5 virtual machines in 2 regions using defined virtual network and storage account
+         *  - Create a traffic manager to route traffic across the virtual machines(Wait for Track2 Traffic Manager ready)
+         */
+        private const string Username = "tirekicker";
+        private const string Password = "12NewPA$$w0rd!";
```
Comment (erich-wang) [inline samples/compute/create-virtual-machines-in-parallel/Program.cs]: change password to placeholder

File: samples/Helpers/Utilities.cs (hunk 1 above)
Comment (HarveyLink) [reply samples/Helpers/Utilities.cs]: Done

File: samples/compute/create-virtual-machines-in-parallel/Program.cs (hunk 2 above)
Comment (HarveyLink) [reply samples/compute/create-virtual-machines-in-parallel/Program.cs]: Done

**Pull request merged**